backend/api/publishing.py

```
@@ -107,61 +311,234 @@ async def save_scenario_draft(
+            scenario.rubric_title = actual_ai_result.get("rubric_title")
+            scenario.rubric_criteria = actual_ai_result.get("rubric_criteria")
+            scenario.rubric_performance_levels = actual_ai_result.get("rubric_performance_levels")
+            scenario.grading_prompt = actual_ai_result.get("grading_prompt")
+            
+            # Preserve the current status when saving (don't force to draft)
+            # Only set to draft if it's a brand new scenario (no existing status)
+            if not scenario.status or scenario.status == "":
+                scenario.status = "draft"
+                scenario.is_draft = True
+                scenario.is_public = False
+                debug_log(f"Setting new scenario status to draft")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)